modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Posts/PostAppService.cs

```diff
@@ -68,10 +68,28 @@ public async Task<ListResultDto<PostWithDetailsDto>> GetListByBlogIdAndTagName(G
                     }
                 }
             }
-
+            
             return new ListResultDto<PostWithDetailsDto>(postDtos);
         }
 
+        public async Task<ListResultDto<PostDto>> GetOrderedListPostsByTime()
+        {
+            var posts = (await _postRepository.GetListAsync()).OrderByDescending(x => x.CreationTime).ToList();
```

Comment:
Why not ordering in the database? Also, getting all posts (without even a blogId filter) seems wrong to me.